src/python/pants/reporting/zipkin_reporter.py

```diff
@@ -15,6 +15,7 @@
 
 logger = logging.getLogger(__name__)
 
+NUM_NANOSECS_IN_SEC = 1000000000.0
```

Comment:
The `_IN_SEC` suffix here looks like we're describing the unit of this variable, which is a bit confusing. Maybe `NANOSECONDS_PER_SECOND` or something?

Reply:
It might be cleaner just to define a const NANOSECOND = 10^-9 then you would multiple instead of divide.

Reply:
I kind of like Stu's suggestion of NANOSECONDS_PER_SECONDS a bit better because the name is explicit. Will go with it.